### PR TITLE
Update Set-PolicyConfig.md

### PR DESCRIPTION
**Pull request by olangness**
Changing the cmdlet name as well as adding a descirption

### CODE REVIEW

**Comment by olangness** [comment]
@samschan-ms Can you add yourself as a reviewer

**Comment by chrisda** [comment]
@olangness, we shouldn't publish this until the change is available to ~50% WW minumum.

**Comment by olangness** [comment]
@chrisda This has been available worldwide for over a year

**Comment by chrisda** [comment]
@olangness, I checked in two different M365 E5 no SDF no flighting tenants (one really old; one really new) where the account is added to a custom role group with all available Email & collaboration roles assigned, and this is what I see in Security & Compliance PowerShell:

 ```powershell
PS C:\Users\jdoe> (Get-Command Set-PolicyConfig).parameters.keys | Sort-Object
CaseHoldPolicyLimit
ClassificationScheme
ComplianceUrl
Confirm
Debug
DlpAppGroups
DlpAppGroupsPsws
DlpNetworkShareGroups
DlpPrinterGroups
DlpRemovableMediaGroups
DocumentIsUnsupportedSeverity
EnableAdvancedRuleBuilder
EnableLabelCoauth
EnableSpoAipMigration
EndpointDlpGlobalSettings
EndpointDlpGlobalSettingsPsws
ErrorAction
ErrorVariable
ExtendTeamsDlpPoliciesToSharePointOneDrive
Identity
InformationAction
InformationBarrierMode
InformationBarrierPeopleSearchRestriction
InformationVariable
IsDlpSimulationOptedIn
OnPremisesWorkload
OutBuffer
OutVariable
PipelineVariable
ProcessingLimitExceededSeverity
PurviewLabelConsent
ReservedForFutureUse
RetentionForwardCrawl
RuleErrorAction
SenderAddressLocation
SiteGroups
SiteGroupsPsws
Verbose
WarningAction
WarningVariable
WhatIf
```

**Comment by samschan-ms** [comment]
> ```powershell
> ExtendTeamsDlpPoliciesToSharePointOneDrive
> ```

Yes @chrisda , doc update is for the description of ExtendTeamsDlpPoliciesToSharePointOneDrive

**Comment by chrisda** [comment]
@samschan-ms, I believe your statement, but this PR seeks to replace the name of the ExtendTeamsDlpPoliciesToSharePointOneDrive parameter that's currently available (whether it's supposed to be or not) with the name ExtendTeamsDlpToSpoOdbConsent, which isn't available to me in my ordinary E5 tenants.

If the intent is to state that ExtendTeamsDlpPoliciesToSharePointOneDrive doesn't function or shouldn't be available on Set-PolicyConfig, then this PR should seek to modify the parameter **description** (only) to say:

> This parameter is reserved for internal Microsoft use.

File: exchange/exchange-ps/exchange/Set-PolicyConfig.md [hunk 1]
```diff
@@ -358,8 +358,9 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ExtendTeamsDlpPoliciesToSharePointOneDrive
-{{ Fill ExtendTeamsDlpPoliciesToSharePointOneDrive Description }}
+### -ExtendTeamsDlpToSpoOdbConsent
+
+This flag is to enable Teams DLP Policy to automatically extend protection to the content stored in OneDrive shared in 1:1/n chats and content stored in SharePoint associated with Teams teams shared through channel chats.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-PolicyConfig.md:363]:
```suggestion
The ExtendTeamsDlpToSpoOdbConsent parameter enables Teams DLP policy to automatically extend protection to the content stored in OneDrive shared in 1:1 chats and content stored in SharePoint associated with Teams teams shared through channel chats.
```

**Comment by samschan-ms** [comment]
@olangness can you modify this PR? remove the change in361?  there is no need to update the parameter. it should simply be an update to the description.

**Comment by chrisda** [comment]
@samschan-ms, I took the liberty of modifying the parameter name as you described. Is it OK?

**Comment by samschan-ms** [comment]
thank you @chrisda

**Pull request merged**